layouts/community/ortho_4x12/symbolic/keymap.c

```diff
@@ -155,11 +220,16 @@ bool process_record_user(uint16_t keycode, keyrecord_t *record) {
   switch (keycode) {
 
 
-    case L_LOWER : return LayerSwitch( record, &l_lower, r_lower, _LOWER ); break;
-    case R_LOWER : return LayerSwitch( record, &r_lower, l_lower, _LOWER ); break;
-    case L_RAISE : return LayerSwitch( record, &l_raise, r_raise, _RAISE ); break;
-    case R_RAISE : return LayerSwitch( record, &r_raise, l_raise, _RAISE ); break;
+    case L_LOWER : return LowerSwitch( record, &l_lower, r_lower ); break;
```

Comment:
I'm not sure that all of this is needed. 

Unless I'm mistaken, you could replace all of this with `MO` keycodes, and:
```c
uint32_t layer_state_set_user(uint32_t state) {
  return update_tri_layer_state(state, _LOWER, _RAISE, _NEUTRAL);
}
```

https://docs.qmk.fm/#/ref_functions?id=update_tri_layer_statestate-x-y-z

Reply:
@drashna In this keymap, I assigned same layer button on both hands. It could predict a situlation that same layer buttons pressed at the same time. If both `LOWER` pressed and then one side released, `LOWER` layer is off although one side `LOWER` have been pressed. This code protect this situlation.

Reply:
Okay, I wasn't entirely sure what you were trying to accomplish here, but that makes more sense.